[2020-10-16 09:52:45,420] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: unscheduled.fetch_events 2020-10-16T09:52:37.821518+00:00 [queued]>
[2020-10-16 09:52:45,425] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: unscheduled.fetch_events 2020-10-16T09:52:37.821518+00:00 [queued]>
[2020-10-16 09:52:45,425] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2020-10-16 09:52:45,425] {taskinstance.py:839} INFO - Starting attempt 1 of 1
[2020-10-16 09:52:45,425] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2020-10-16 09:52:45,434] {taskinstance.py:859} INFO - Executing <Task(BashOperator): fetch_events> on 2020-10-16T09:52:37.821518+00:00
[2020-10-16 09:52:45,434] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'unscheduled', 'fetch_events', '2020-10-16T09:52:37.821518+00:00', '--job_id', '50', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/data_piplines_book/chapter3/unscheduled.py', '--cfg_path', '/tmp/tmpfd3dyqvo']
[2020-10-16 09:52:45,781] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events [2020-10-16 09:52:45,781] {settings.py:213} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=104086
[2020-10-16 09:52:45,872] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events [2020-10-16 09:52:45,872] {__init__.py:51} INFO - Using executor LocalExecutor
[2020-10-16 09:52:46,077] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events [2020-10-16 09:52:46,076] {dagbag.py:90} INFO - Filling up the DagBag from /usr/local/airflow/dags/data_piplines_book/chapter3/unscheduled.py
[2020-10-16 09:52:46,196] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events [2020-10-16 09:52:46,195] {cli.py:516} INFO - Running <TaskInstance: unscheduled.fetch_events 2020-10-16T09:52:37.821518+00:00 [running]> on host 5501f7880404
[2020-10-16 09:52:46,205] {bash_operator.py:81} INFO - Tmp dir root location: 
 /tmp
[2020-10-16 09:52:46,205] {bash_operator.py:91} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=unscheduled
AIRFLOW_CTX_TASK_ID=fetch_events
AIRFLOW_CTX_EXECUTION_DATE=2020-10-16T09:52:37.821518+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2020-10-16T09:52:37.821518+00:00
[2020-10-16 09:52:46,205] {bash_operator.py:105} INFO - Temporary script location: /tmp/airflowtmphnsniz0r/fetch_eventst47dgmu4
[2020-10-16 09:52:46,205] {bash_operator.py:115} INFO - Running command: mkdir -p /data/events && curl -o /data/events.json https://0.0.0.0:5000/events
[2020-10-16 09:52:46,208] {bash_operator.py:124} INFO - Output:
[2020-10-16 09:52:46,209] {bash_operator.py:128} INFO - bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
[2020-10-16 09:52:46,213] {bash_operator.py:128} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2020-10-16 09:52:46,213] {bash_operator.py:128} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2020-10-16 09:52:46,213] {bash_operator.py:128} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed to connect to 0.0.0.0 port 5000: Connection refused
[2020-10-16 09:52:46,213] {bash_operator.py:132} INFO - Command exited with return code 7
[2020-10-16 09:52:46,219] {taskinstance.py:1051} ERROR - Bash command failed
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/bash_operator.py", line 136, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2020-10-16 09:52:46,220] {taskinstance.py:1082} INFO - Marking task as FAILED.
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events Traceback (most recent call last):
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events   File "/usr/local/bin/airflow", line 32, in <module>
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events     args.func(args)
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events     return f(*args, **kwargs)
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events     _run(args, dag, ti)
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events     pool=args.pool,
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events     return func(*args, **kwargs)
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2020-10-16 09:52:46,230] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events     result = task_copy.execute(context=context)
[2020-10-16 09:52:46,231] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events   File "/usr/local/lib/python3.7/site-packages/airflow/operators/bash_operator.py", line 136, in execute
[2020-10-16 09:52:46,231] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events     raise AirflowException("Bash command failed")
[2020-10-16 09:52:46,231] {base_task_runner.py:115} INFO - Job 50: Subtask fetch_events airflow.exceptions.AirflowException: Bash command failed
[2020-10-16 09:52:50,452] {logging_mixin.py:95} INFO - [[34m2020-10-16 09:52:50,451[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
